lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -10,6 +19,23 @@
         },
         dataType: 'json'
       });
+
```

Comment:
@chrisndodge -- we're wondering if you have a suggestion of a reasonable location to move this Javascript?  There doesn't seem to be a dedicated spot for containing functions like this at the moment.   I suppose we could add a new facebook.js file to the mix -- thoughts on that?